src/ui/public/saved_objects/saved_objects_client.test.ts

```diff
@@ -19,79 +19,41 @@
 
 jest.mock('ui/kfetch', () => ({}));
 
-import sinon from 'sinon';
+// @ts-ignore
```

Comment:
Expect.js typings are already used in `x-pack`, so wouldn't it make sense to also use them here?